CHANGELOG.rst

```diff
@@ -2,9 +2,14 @@
 Release Notes
 =============
 
-Unreleased
+1.18.0
 ================
 
+- AP-2106 Add new GPG keys for APT and RPM repositories signature rotation (#44)
+- Restore usage of underscores in options as it would require change at several places and would probably bring confusion (#52)
+- SLES 11 install script support (#51)
```

Comment:
Technically, SLES 11 was supported before, but we broke support in an earlier version, so I'd change this to:
```suggestion
- Fix install script on SLES 11 (#51, #52)
```